src/erlcloud_aws.erl

```diff
@@ -511,3 +518,17 @@ get_filtered_statuses(ServiceNames, Statuses) ->
             ServiceNames)
         end,
     Statuses).
+
+-spec is_throttling_error_response(aws_request()) -> true | false.
+is_throttling_error_response(RequestResponse) ->
+    #aws_request{
+         response_type = error,
+         error_type = aws,
+         response_body = RespBody} = RequestResponse,
+  
+    case re:run(RespBody, <<"Throttling">>) of
```

Comment:
i have no better ideas given the amount of case matching we need to do to cover them all